pages/contact.html

```diff
@@ -89,102 +90,92 @@
                 </div>
                 <div>
-                  <strong class="info">Email:</strong> <a href="mailto:example@mail.com">
-                    example@mail.com</a>
+                  <strong class="info">Email:</strong> <a href="mailto:acmuwbothell@gmail.com">
+                    acmuwbothell@gmail.com</a>
                 </div>
               </div>
             </div>
             <div class="column3">
               <h3 class="header follow">Follow us </h3>
-              <a href="#facebook" class="facebook" title="facebook"><span class="fa fa-facebook"></span></a>
-              <a href="#twitter" class="twitter" title="twitter"><span class="fa fa-twitter"></span></a>
-              <a href="#linkedin" class="linkedin" title="linkedin"><span class="fa fa-linkedin"></span></a>
-              <a href="#instagram" class="instagram" title="instagram"><span class="fa fa-instagram"></span></a>
+              <a href="https://www.facebook.com/uwbacm" target="_blank" class="facebook" title="facebook"><span
+                  class="fa fa-facebook-f"></span></a>
+              <a href="https://twitter.com/hacksuwb"  target="_blank" class="twitter" title="twitter"><span
+                  class="fa fa-twitter"></span></a>
+              <a href="https://www.linkedin.com/organization-guest/company/uwb-acm-club" target="_blank"
+                class="linkedin" title="linkedin"><span class="fa fa-linkedin"></span></a>
+              <a href="https://discord.gg/YBrxBTg" target="_blank" class="discord" title="discord"><span
+                  class="fab fa-discord"></span></a>
+              <a href="https://www.instagram.com/uwb_acm/" target="_blank" class="instagram" title="instagram"><span
+                  class="fa fa-instagram"></span></a>
             </div>
           </div>
-          <!-- map iframe-->
           <div class="col-md-6 map mt-lg-0 mt-4">
-            <iframe
-              src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d387190.2895687731!2d-74.26055986835598!3d40.697668402590374!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x89c24fa5d33f083b%3A0xc80b8f06e177fe62!2sNew+York%2C+NY%2C+USA!5e0!3m2!1sen!2sin!4v1562582305883!5m2!1sen!2sin"
-              frameborder="0" style="border:0" allowfullscreen=""></iframe>
+              <img src="../assets/images/backgrounds/home-background.jpg" class="contact-image">
           </div>
-          <!-- //map iframe -->
         </div>
+        <div class="col-lg-8 offset-lg-2 col-md-12 col-sm-12 px-lg-3 px-0">
+          <br>
+        <h5 class="section-title">Interested in working with us?</h5>
+        <p class= "text-center"> Are you an alum who wants to participate in our events? Email us to see how you can make difference with the UWB ACM. </p>
+          </div>
       </div>
     </div>
   </section>
-
   <section class="contact-form">
     <div class="container">
       <div class="contacts12-main">
```

Comment:
The `<section>`, and top 2 `<div>` are not being closed. If you look at the website you'll see that the footer is broken.